src/jvm/main/org/jetbrains/kotlinx/lincheck/LinChecker.kt

```diff
@@ -197,4 +203,6 @@ fun <O : Options<O, *>> O.check(testClass: Class<*>) = LinChecker.check(testClas
  */
 fun <O : Options<O, *>> O.check(testClass: KClass<*>) = this.check(testClass.java)
 
-internal fun <O : Options<O, *>> O.checkImpl(testClass: Class<*>) = LinChecker(testClass, this).checkImpl()
\ No newline at end of file
+internal fun <O : Options<O, *>> O.checkImpl(testClass: Class<*>) = LinChecker(testClass, this).checkImpl()
+
+internal const val NO_OPERATION_ERROR_MESSAGE = "You should provide at least one operation to check. Please see the official guide: https://kotlinlang.org/docs/introduction.html"
```

Comment:
You must specify at least one operation to test. Please refer to the user guide: https://kotlinlang.org/docs/introduction.html